docs/docs/index.md

```diff
@@ -1,143 +1,134 @@
+## Installation
 
-This documentation covers three methods:
+--8<-- "snippets/install.md"
 
-- [Installing Jupyter within a virtual environment](#installing-jupyter-within-a-virtual-environment)
-- [Installing `rsconnect-jupyter` to Jupyter running on Posit Workbench](#installing-to-jupyter-running-on-posit-workbench)
-- [Installation in JupyterHub](#installing-in-jupyterhub)
+## Runtime Environment Configuration
 
-Please navigate to the installation section below that is best for your environment.
+### Localhost (Your Computer)
 
-### Installing Jupyter within a virtual environment
+For localhost installation, a Python virtual environment is recommend to isolate runtime dependencies. There are various Python virtual environments avaiable. The following tutorial covers a few of them.
 
-To install and use Jupyter within a virtual environment using
-`virtualenv`, follow the procedures shown below or learn more using the
-[Virtualenv](https://virtualenv.pypa.io/en/latest/) documentation.
+#### Conda
 
-- These commands create and activate a `virtualenv` at `/my/path`:
-  <div class="code-title">Terminal</div>
-  ```bash
-  $ pip install virtualenv
-  virtualenv /my/path
-  source /my/path/bin/activate
-  ```
+Install `rsconnect-jupyter` from [Conda Forge](https://conda-forge.org).
 
-!!! tip
-    Running `source /my/path/bin/activate` activates the virtual environment. While the `virtualenv` is active, Python-related commands like `python`, `pip`, and `jupyter` will use to copies located inside the virtual environment. You can check which copy of `python` you're using by running `which python`.
+<div class="code-title">Terminal</div>
+```shell
+conda create --name rsconnect-jupyter
+conda activate rsconnect-jupyter
+conda install -c conda-forge jupyter rsconnect-jupyter
+```
 
-- Install Jupyter inside the `virtualenv`:
-  <div class="code-title">Terminal</div>
-  ```bash
-  $ pip install jupyter
-  ```
+Next, following the [installation guide](#installation).
 
-- Install rsconnect-jupyter with your virtual environment active to install and activate the plugin for that copy of Jupyter:
+!!! Tip
+    When creating a _Conda_ virtual environment, a specific Python version may be specified. Create your virtual environment with a Python environment that is avaiable on your Posit Connect server.
 
-    --8<-- "snippets/python_pkg.md"
+    <div class="code-title">Terminal</div>
+    ```shell
+    conda create --name rsconnect-jupyter python=3.8
+    ```
 
-!!! important
-    Be sure to run Jupyter from this virtual environment, not from
-    another installation, or the `rsconnect-jupyter` extension will
-    not be available. To do so, you will need to activate the virtual
-    environment in each new terminal session before you run `jupyter`.
+#### Python Virtual Environment (virtualenv)
 
----
+<div class="code-title">Terminal</div>
+```shell
+python -m pip install virtualenv
+python -m virtualenv .venv
```

Comment:
We should probably recommend `python -m venv` unless there's a specific need to use `virtualenv`.